globals.json

```diff
@@ -437,27 +443,28 @@
 	"worker": {
 		"importScripts": true,
 		"postMessage": true,
-		"self": true
+		"self": true,
+		"FileReaderSync": true
 	},
 	"node": {
 		"__filename": false,
 		"__dirname": false,
 		"arguments": false,
-		"Buffer": false,
+		"Buffer": true,
```

Comment:
According to the original issue, developers overwrite Buffer, console and so on in some cases.
https://github.com/jshint/jshint/pull/1426

I think that it is not good thing. So I will remove the changes for "node" environment.